kotlinx-coroutines-core/jvm/resources/META-INF/proguard/coroutines.pro

```diff
@@ -3,12 +3,12 @@
 -keepnames class kotlinx.coroutines.CoroutineExceptionHandler {}
 
 # Most of volatile fields are updated with AFU and should not be mangled
--keepclassmembernames class kotlinx.** {
+-keepclassmembers class kotlinx.** {
```

Comment:
While we're here it would be nice to scope these to the correct package rather than such a top-level one which encompasses many libraries.

Reply:
Good call, thanks!